fiware-region-sanity-tests/resources/scripts/jenkins.sh

```diff
@@ -251,6 +251,12 @@ if [ -z "$FIHEALTH_ADAPTER_URL" -o -z "$FIHEALTH_CB_URL" ]; then
 	exit 3
 fi
 
+# Check FIHealth environment variables for Jenkins
+if [ -z "$JENKINS_USER" -o -z "$JENKINS_PASSWORD" ]; then
+	printf "Either Jenkins user or Jenkins password not specified\n" 1>&2
+	exit 3
+fi
```

Comment:
As these new environment variables are compulsory, please include them at the initial comment of the script, together with a brief explanation.